src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/destination/Chime.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.model.destination
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.elasticapi.string
+import org.elasticsearch.common.Strings
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.common.xcontent.XContentFactory
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import org.elasticsearch.common.xcontent.XContentType
+import java.io.IOException
+import java.lang.IllegalStateException
+
+/**
+ * A value object that represents a Chime message. Chime message will be
+ * submitted to the Chime destination
+ *
+ * Temporary import from alerting, this will be removed once we pull notifications out of
+ * alerting so all plugins can consume and use.
+ */
+data class Chime(val url: String) : ToXContent {
+
+    init {
+        require(!Strings.isNullOrEmpty(url)) { "URL is null or empty" }
```

Comment:
will this throw correct exception message if the `url` is null? I think it would throw a generic exception from Kotlin giving exception message about how we can assign `null` to `String`. Lets use convention we are using in ISM code where we check before we pass into the constructor.

Reply:
This was copied from Alerting (temporary fix until Notifications is made standalone). I changed url to be nullable so that we can throw the specific message here.